addons/sourcemod/scripting/szf/sound.sp

```diff
@@ -203,16 +203,18 @@ char g_strSoundSurivourWin[][PLATFORM_MAX_PATH] =
 //Special infected spawned
 char g_strSoundSpawnInfected[][PLATFORM_MAX_PATH] =
 {
-	""										//no special infected
-	,""										//tank
+	"ui/null.wav"							//no special infected
+	,"ui/null.wav"							//tank
 	,"left4fortress/boomerbacterias.mp3"	//boomer
 	,"left4fortress/chargerbacterias.mp3"	//charger
-	,""										//kingpin, used to use smoker's bacteria (need new sound for kingpin)
+	,"ui/null.wav"							//kingpin, used to use smoker's bacteria (need new sound for kingpin)
```

Comment:
Just found out myself i'm forced to try download null sound, stock TF2 sounds shouldn't really be in downloads

Reply:
right, just removed it then.

Reply:
You should be able to put stock tf2 sounds into download without any problem. It works fine in Slender Fortress for instance.
I don't think ui/null.wav exist anymore which is probably the issue. You should try `vo/null.mp3`

Reply:
yeah, `ui/null.wav` isn't there (and maybe it never was), oops. One thing is that if I use `vo/null.mp3`, zombies would play a voiceline when spawning because it has `vo/`in it. I found out that `misc/null.wav` does exist (and may have been what I was trying to use), and that works. Thanks Ben!

Reply:
My pleasure.